metron-deployment/README.md

```diff
@@ -133,6 +133,8 @@ Notably, the URL for the GeoIP database that is preloaded (and is prefilled by d
 
 After installation, a custom action is available in Ambari (where stop / start services are) to install Elasticsearch templates.  Similar to this, a custom Kibana action to Load Template is available.
 
+Another custom action is available in Ambari to import Zeppelin dashboards.
```

Comment:
Should we link from this mention here to the more detailed information that you have in `metron-platform/metron-indexing/README.md`?